Sources/muterCore/MutationOperators/ChangeLogicalConnectorOperator.swift

```diff
@@ -6,7 +6,7 @@ extension ChangeLogicalConnectorOperator {
             super.init(positionToMutate: positionToMutate)
             oppositeOperatorMapping = [
                 "||": "&&",
-                "&&": "||",
+                "&&": "||"
```

Comment:
boooooo 😆

(I think trailing commas on the last line of multiline collections are better, because they make for cleaner diffs if you add or remove an item at the end later. I believe this strongly enough that I usually turn on the SwiftLint rule for it.)